assets/i18n/id.json

```diff
@@ -0,0 +1,262 @@
+    "dgc-exchanges-modal-fah-faq": "<a class=\"btn btn-sm btn-outline-secondary\" target=\"_blank\" href=\"https://www.doggeinfah.com\" target=\"_blank\">Folding@Home (FaH)</a> adalah proyek komputasi terdistribusi. Kamu bisa bergabung dengan tim dogefolders, dengan membawa kekuatan komputasimu untuk membantu menemukan pengobatan penyakit dan dapatkan beberapa Doge atas usahamu.", 
+    "dgc-exchanges-modal-faucet-sites": "Berikut adalah sumber informasi untuk mulai melipat:", 
+    "dgc-exchanges-modal-indogewetrust": "InDogeWeTrust.com", 
+    "dgc-exchanges-modal-indogewetrust-desc": " - Sama seperti Dogefaucet.", 
+    "dgc-exchanges-modal-dogechat": "Dogechat.net", 
+    "dgc-exchanges-modal-dogechat-desc": " - Mengobrol dengan komunitas untuk mendapatkan Dogecoin gratis.", 
+    "dgc-exchanges-modal-exchanges": "Kamu juga bisa membelinya atau bertukar dengan uang konvensional. Ini adalah cara yang mudah untuk mendapatkan Dogecoin.", 
+    "dgc-exchanges-modal-list": "Berikut beberapa layanan penukaran Dogecoin yang terpercaya:", 
+    "dgc-exchanges-modal-weselldoge": "WeSellDoge", 
+    "dgc-exchanges-modal-weselldoge-desc": " (Paypal)", 
+    "dgc-exchanges-modal-celery": "Celery", 
+    "dgc-exchanges-modal-celery-desc": " (USD)", 
+    "dgc-exchanges-modal-anxpro": "ANXPRO", 
+    "dgc-exchanges-modal-anxpro-desc": " (USD, EUR, GBP, AUD, CAD, HKD, SGD, NZD)", 
+    "dgc-exchanges-modal-localdogecoin": "LocalDogecoin",
+    "dgc-exchanges-modal-localdogecoin-desc": " (Sebuah website untuk membeli Dogecoin lokal.)",
+    "dgc-exchanges-modal-mining": "Kamu juga bisa mendapatkan Dogecoin dengan \"menambang\"nya. Penambangan ialah saat kamu menggunakan komputer untuk memproses transaksi Dogecoin orang-orang, dan sebagai imbalannya kamu bisa mendapatkan Dogecoin.", 
+    "dgc-exchanges-modal-mining-howto": "Penambangan hanya ditujukan untuk pengguna tingkat lanjut, karena untuk melakukannya membutuhkan banyak pengetahuan teknis. Jika kamu tertarik, <a class=\"btn btn-sm btn-outline-secondary\" href=\"https://dogecointutorial.com/mining.html\" target=\"_blank\">baca selengkapnya disini</a>.", 
+    "dgc-exchanges-modal-disclaimer": "<strong>Catatan:</ strong> Faucet dan penukaran daring tersebut tidak dimiliki atau dikelola oleh Dogecoin. Sementara kami puas dan mempercayainya, karena pelayanannya dapat diandalkan - mohon berhati-hati dan resiko ditanggung sendiri.", 
+    "dgc-stores-modal-title": "TOKO:", 
+    "dgc-stores-modal-shop-online": "Kamu bisa berbelanja daring dengan Dogecoin.", 
+    "dgc-stores-modal-sale": "Dogecoin tergolong aman selama checkout, karena kamu tidak memerlukan informasi bank yang sensitif! Cukup kirim Dogecoinmu ke alamat yang ditentukan oleh pedagang, kemudian barangmu siap untuk diproses dan dikirim dalam sekejap!", 
```

Comment:
```
Cukup kirim Dogecoinmu ke alamat yang ditentukan oleh penjual
```

I think it's more common for people to use "penjual" instead of "pedagang" to describe online seller

Reply:
```suggestion
    "dgc-stores-modal-sale": "Dogecoin tergolong aman selama checkout, karena kamu tidak memerlukan informasi bank yang sensitif! Cukup kirim Dogecoinmu ke alamat yang ditentukan oleh penjual, kemudian barangmu siap untuk diproses dan dikirim dalam sekejap!", 
```